README.md

```diff
@@ -117,12 +46,12 @@ Assumptions made about production:
 - Apache httpd server is used to serve app using mod_wsgi.
 - All static files are served from Apache using a redirect from url `/static/` to a static documents folder. This folder is created using `python manage.py collectstatic`.  This is not necessary in development.
 
-##License and Copyright
-All source code is covered by the MIT license.  This license is located in the LICENSE.txt file at the root of the project.
+## License and Copyright
+All source code is covered by the MIT license.  This license can be found [here](https://github.com/jakeharding/repo-health/blob/master/LICENSE.txt).
 
```

Comment:
Should we put an actual copyright here?  Something like `© 2017 Jake Harding and Benji Parish`?